flake.nix

```diff
@@ -77,6 +78,7 @@
         # The Resonate server
         resonate = pkgs.buildGoApplication rec {
           pname = "resonate";
+          inherit shortRev;
```

Comment:
You can remove this line (you don't need to inherit this value into the derivation function to use it :smile:)